string grounded_predicate
---
string verbalization
